media-and-outreach/chaosscast.md

```diff
@@ -17,46 +17,26 @@ The **host** is responsible for the making the initial connection to potential g
 1. Georg Link
 2. Samantha Venia Logan
```

Comment:
We might check with Venia to see if she is still interested in participating.

Reply:
I am still intrested yes, but given the imbalance that I seem to have in CHAOSS in general, I'm concerned with taking on any form of leadership role. I'm good with being a panelist and with organizing events related to community, metrics building, and company interfacing with both, but I made the choice to step back from everything else after the communication group became overwhelming for me. 

I'm not sure I want any further involvement than this.